includes/agentic-commerce/class-wc-stripe-agentic-checkout-session.php

```
@@ -130,10 +133,19 @@ public function get_billing_phone(): ?string {
+	public function get_billing_address(): WC_Stripe_API_Address {
+		$address = $this->session->customer_details->address ?? null;
+		if ( null === $address ) {
+			throw new Exception(
+				sprintf(
+					'Checkout session %s has no billing address.',
+					$this->get_id()
+				)
+			);
+		}
+		return new WC_Stripe_API_Address( $address );
 	}
```

PHPMD (2.15.0)

[error] The method get_billing_address is not named in camelCase. (undefined)

(CamelCaseMethodName)